aten/src/ATen/native_parse.py

```diff
@@ -211,6 +211,15 @@ def parse_arguments(args, func_variants, declaration, func_return):
     supported_topt_arguments.append(copy.deepcopy(supported_topt_arguments[1]))
     for arg in supported_topt_arguments[2]:
         arg.update({'default': 'c10::nullopt', 'is_nullable': True})
+    # add explicit support for what is needed for tril_indices / triu_indices
```

Comment:
The only difference is the data type, so I'm surprised this needs to be so verbose. But it's not a big deal in my mind with the assumption that this will be deleted anyway.